components/common/posts/PostsFeed.tsx

```diff
@@ -331,6 +333,67 @@ export function RecordEmbedView({
         </div>
       </div>
     )
+  } else if (AppBskyGraphDefs.isListView(embed.record)) {
+    const { did, rkey } = parseAtUri(embed.record.uri) || {}
+    const peekLink = buildBlueSkyAppUrl({
+      did: `${did}`,
+      rkey: `${rkey}`,
+      collection: `lists`,
+    })
+
+    return (
+      <div
+        className={`flex gap-2 pb-2 ${leftPadding} flex-col border-2 border-gray-400 border-dashed my-2 rounded pt-2`}
+      >
+        <div className="flex flex-row gap-1">
+          <ProfileAvatar
+            profile={{
+              avatar: embed.record.avatar,
+              did: embed.record.creator.did,
+              handle: embed.record.name,
+            }}
+            className="w-6 h-6 rounded-full"
+          />
```

Comment:
Would it be worth renaming the `ProfileAvatar` component or extracting out a more generic component from it?  I notice that we're "tricking" it into thinking it's displaying profile information.

Reply:
hmm.. idk, it's minimal lift but I also don't see avatars being shown for anything other than users/profiles so don't think it it matters a lot either way?